README.md

```diff
@@ -5,12 +5,32 @@
 [![Linux CI Tests](https://github.com/Azure-Samples/iot-middleware-freertos-samples/actions/workflows/ci_tests_linux.yml/badge.svg)](https://github.com/Azure-Samples/iot-middleware-freertos-samples/actions/workflows/ci_tests_linux.yml)
 <!-- markdown-link-check-enable-->
 
-## Getting Started
+## Getting Started with Azure IoT middleware for FreeRTOS
 
 The Azure IoT middleware for FreeRTOS simplifies the connection of devices running FreeRTOS to Azure IoT services. It implements a modular approach that brings flexibility to IoT developers by allowing them to bring their own network stack (MQTT, TLS and Socket).
 
 The [Azure IoT middleware for FreeRTOS repo](https://github.com/Azure/azure-iot-middleware-freertos) has the core functionalities of the middleware and no external dependencies, however to implement working samples, we need to bring a network stack which is the objective of this repo. 
 
+We have two sets of samples:
+
+- **IoT Central**: These samples leverage DPS (Device Provisioning), Azure IoT Central and Plug and Play. Azure subscription is not required during the initial 7 days of the [Free Trial](https://azure.microsoft.com/en-us/pricing/details/iot-central/#pricing).
+- **IoT Hub**: These samples leverage DPS (Device Provisioning), Azure IoT Hub and Plug and Play.
+
+
+### IoT Central Samples:
+
+Below you will find samples for development kits showing how to use the Azure IoT middleware for FreeRTOS. The following development kits are currently supported:
+
+<!-- ESP32 SaaS sample to be released soon
+* ESPRESSIF:
+  * [ESP32](demos/projects/ESPRESSIF/esp32/gsg.md)
+-->
+
+* STMicroelectronics:
+  * [B-L475E-IOT01A](demos/projects/ST/b-l475e-iot01a/gsg.md)
```

Comment:
I wonder if instead of calling these md files gsg.md, we should name them getting-started-guide.md or similar, trying to improve discoverability. What do you think?

Reply:
it would be better indeed, but this arrangement is temporary. This gsg.md file will be moved to the Microsoft Docs page and we'll point people to that page instead.